client/packages/requisitions/src/RequestRequisition/DetailView/columns.ts

```diff
@@ -250,7 +250,7 @@ export const useRequestColumns = () => {
       accessor: ({ rowData }) => rowData.linkedRequisitionLine?.approvalComment,
     });
   }
-  columnDefinitions.push(GenericColumnKey.Selection);
+  columnDefinitions.unshift(GenericColumnKey.Selection);
```

Comment:
Thanks for changes Noel! One last little request from me - I do think just putting `GenericColumnKey.Selection` as the first array item, as you have in the other places, would be clearer than the unshifts 🙏

Reply:
Fixed here 3b8bb42be443cadfdef8514732cf9c3c9861af50 :)